app/backend/src/couchers/models.py

```diff
@@ -598,6 +598,23 @@ class Email(Base):
     html = Column(String, nullable=False)
 
 
+class SMS(Base):
+    """
+    Table of all sent SMSs for debugging purposes, etc.
+    """
+
+    __tablename__ = "smss"
+
+    id = Column(String, primary_key=True)
+
+    # timezone should always be UTC
+    time = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
+
+    sender = Column(String, nullable=False)
```

Comment:
sender_user_id maybe you meant?

Reply:
This is the SMS sender ID that the SMS is sent from. I think it's a phone number.